dolphinscheduler-data-quality/src/test/java/org/apache/dolphinscheduler/data/quality/flow/reader/ReaderFactoryTest.java

```
@@ -0,0 +1,72 @@
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.data.quality.flow.reader;
+
+import static org.apache.dolphinscheduler.data.quality.Constants.DATABASE;
+import static org.apache.dolphinscheduler.data.quality.Constants.DRIVER;
+import static org.apache.dolphinscheduler.data.quality.Constants.PASSWORD;
+import static org.apache.dolphinscheduler.data.quality.Constants.TABLE;
+import static org.apache.dolphinscheduler.data.quality.Constants.URL;
+import static org.apache.dolphinscheduler.data.quality.Constants.USER;
+
+import org.apache.dolphinscheduler.data.quality.config.ReaderConfig;
+import org.apache.dolphinscheduler.data.quality.flow.batch.BatchReader;
+import org.apache.dolphinscheduler.data.quality.flow.batch.reader.ReaderFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * ConnectorFactoryTest
+ */
+public class ReaderFactoryTest {
+
+    @Test
+    public void testConnectorGenerate() {
+
+        List<ReaderConfig> readerConfigs = new ArrayList<>();
+        ReaderConfig readerConfig = new ReaderConfig();
+        readerConfig.setType("JDBC");
+        Map<String,Object> config = new HashMap<>();
+        config.put(DATABASE,"test");
+        config.put(TABLE,"test1");
+        config.put(URL,"jdbc:mysql://localhost:3306/test");
+        config.put(USER,"test");
+        config.put(PASSWORD,"123456");
+        config.put(DRIVER,"com.mysql.jdbc.Driver");
+        readerConfig.setConfig(config);
+        readerConfigs.add(readerConfig);
+
+        int flag = 0;
+        try {
+            List<BatchReader> readers = ReaderFactory.getInstance().getReaders(null,readerConfigs);
+            if (readers != null && readers.size() >= 1) {
+                flag = 1;
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        Assert.assertEquals(1,flag);
```

No need to write this kind of codes, just remove the `try-catch` and let it throw the exception to make the test fail